Content.Server/GameObjects/Components/PDA/PDAComponent.cs

```diff
@@ -293,6 +293,26 @@ protected override void Activate(IEntity user, PDAComponent component)
             }
         }
 
+        public sealed class ToggleFlashlightVerb : Verb<PDAComponent>
+        {
+            protected override void GetData(IEntity user, PDAComponent component, VerbData data)
+            {
+                if (!ActionBlockerSystem.CanInteract(user))
+                {
+                    data.Visibility = VerbVisibility.Invisible;
+                    return;
+                }
+
+                data.Text = Loc.GetString("Toggle flashlight");
+                data.Visibility = VerbVisibility.Visible;
```

Comment:
I don't think you actually need to set it to visible (probably defaults to that)